File: 00_Tasks/01a_File_formats_bonanza/javascript/read_json_file.js
import fs from 'fs'

fs.readFile('../me.json', 'utf8', (err, data) => {
    if (err) {
        console.error('Error reading the file:', err);
        return;
    }
    
    const profile = JSON.parse(data);
    console.log(profile);
    console.log('Name:', profile.name);
    console.log('Age:', profile.age);
    console.log('Hobbies:', profile.hobbies.join(', '));
});

File: 00_Tasks/03a_Data_parsing_server/python_server/read_json_file.py
import json

def read_json():
    with open('../files/me.json', 'r') as file:
        data = json.load(file)
    return data

# print(data)
# print('Name:', data['name'])
# print('Age:', data['age'])
# print('Hobbies:', ', '.join(data['hobbies']))